terraform/aptos-node/aws/README.md

```diff
@@ -0,0 +1,181 @@
+## Run on AWS
+This guide assumes you already have AWS account setup.
+
+Install pre-requisites if needed:
+
+   * Terraform 1.1.7: https://www.terraform.io/downloads.html
+   * Kubernetes cli: https://kubernetes.io/docs/tasks/tools/
+   * AWS cli: https://aws.amazon.com/cli/
+
+1. Create a working directory for your configuration.
+
+    * Choose a workspace name e.g. `testnet`. Note: this defines terraform workspace name, which in turn is used to form resource names.
+    ```
+    $ export WORKSPACE=testnet
+    ```
+
+    * Create a directory for the workspace
+    ```
+    $ mkdir -p ~/$WORKSPACE
+    ```
+2. Create a S3 storage bucket for storing the Terraform state on AWS. 
+
+3. Create Terraform file called `main.tf` in your working directory:
+  ```
+  $ cd ~/$WORKSPACE
+  $ touch main.tf
+  ```
+
+4. Modify `main.tf` file to configure Terraform, and create fullnode from Terraform module. Example content for `main.tf`:
+  ```
+  variable "aws_region" {
```

Comment:
why do we need this variable if we already have `var.region` (https://github.com/aptos-labs/aptos-core/blob/main/terraform/aptos-node/aws/variables.tf#L1) ?

Reply:
because here you still need to set the region for s3 bucket and the region to use by the module, since there's multiple, better to just define it in one place